ext/node/lib.rs

```diff
@@ -354,7 +355,8 @@ pub fn init_polyfill_ops_and_esm() -> Extension {
     "zlib.ts",
   );
 
-  ops_polyfill(&mut ext_polyfill())
+  ext_polyfill()
+    .ops(deno_ops_polyfill())
     .esm(esm_files)
     .esm_entry_point("ext:deno_node/module_all.ts")
```

Comment:
I believe it's a TODO at this point, but we should be able to specify both `esm()` and `esm_entry_point()` in the `extension!` macro right?